src/RcsbFv/RcsbFv.tsx

```diff
@@ -144,9 +144,16 @@ export class RcsbFv<
     public init(): Promise<void> {
         this.rcsbFvPromise = new Promise<void>((resolve, reject)=>{
             if(!this.mounted && this.boardConfigData != undefined) {
-                const node: HTMLElement|null = document.getElementById(this.elementId);
-                if(node==null)
-                    throw `ERROR: HTML element ${this.elementId} not found`
+                let node: HTMLElement|null;
```

Comment:
Please, initialize `node` as const using a ternary operator and assert it is not null after.

Reply:
👍